packages/server/src/orders/database/OrderSchema.ts

```diff
@@ -0,0 +1,74 @@
+import { Prop, raw, Schema, SchemaFactory } from '@nestjs/mongoose';
+import mongoose, { Document } from 'mongoose';
```

Comment:
Jak zmienisz tego Prop, gdzie jest ref, to wtedy nie trzeba importowac mongoose
```suggestion
import { Document } from 'mongoose';
```